cmd/status/cmdstatus.go

```diff
@@ -164,11 +164,14 @@ func (r *Runner) loadInvFromDisk(cmd *cobra.Command, args []string) (*printer.Pr
 
 	// Based on the inventory template manifest we look up the inventory
 	// from the live state using the inventory client.
-	identifiers, err := invClient.GetClusterObjs(inv)
+	inv, err := invClient.Load(context.TODO(), invInfo)
```

Comment:
Does the cobra.Command have a context we can pass in here?

Reply:
`r.ctx` is used elsewhere

Reply:
Ah yes. We can re-use that. Done.